package.json

```diff
@@ -6,9 +6,7 @@
   "typings": "dist/index",
   "files": [
     "dist/**/*.js",
-    "dist/**/*.d.ts",
-    "setup/*.js",
-    "setup/*.d.ts"
```

Comment:
the setup script should be part of the package to improve usability

Reply:
It's included under dist in `dist/setup/` -- I moved setup under src to avoid the typescript outDir issue mentioned above (the current published version of the package is in a bad state).

Alternatively, the BitPaySetup could be a plain js file and _not_ compiled with typescript. That would make it easier to put under the "setup" directory, a peer of "dist". If BitPaySetup is going to be a .ts file, it should probably be under `dist/setup/` though.

Reply:
@anbugal Let me know if you'd prefer to have BitPaySetup be a plain .js file instead of a .ts file (it looks that might have been the intention of it originally). I can push that change quickly so you can merge this.